### PR TITLE
Incorporate balanced mod 2.0

### PR DESCRIPTION
**Pull request by simon33-2**
Code to add balanced mod 2.0 from triplea-maps/world_war_ii_g40_balanced.

Untested - not sure how this can be tested. Perhaps someone can help with this??

### CODE REVIEW

**Comment by panther2** [comment]
Not an expert myself, but to me it looks like you replace the map-info for the "regular" Global-map with the Balanced-map with his pull request (once this gets merged).

**Comment by simon33-2** [comment]
@panther2 @DanVanAtta 

Not sure I understand the above comment. If the maps are picked up from this file, I'm unaware of a method of testing it other than merging and seeing if it works. Perhaps I can change a pointer somewhere on my computer first.

BTW, yes the idea is that it is additional to Global. With some of the updates, it's a new section of maps not part of the Global orig/Alpha+3/2nd ed/Global 1942 set.

**Comment by DanVanAtta** [comment]
@simon33-2  you've some merge conflicts, you may need to redo your changes from a fresh master, or see if you can do a merge conflict resolution.

To test:
- look to local files, find 'game_engine.properties'
- uncomment this line; https://github.com/triplea-game/triplea/blob/master/game_engine.properties#L6

You'll then use the local copy of triplea_maps.yaml, you can modify that, test, and us it to submit a PR.

**Comment by simon33-2** [comment]
@DanVanAtta @panther2 

Looks like it works now.

Also affected: https://github.com/triplea-maps/Project/issues/36

File: production_config/triplea_maps.yaml [hunk 1]
```diff
@@ -317,6 +317,19 @@
     <br>http://tripleadev.1671093.n2.nabble.com/World-War-II-Global-1940-Original-tp7314662p7314662.html
     <br>
   version: 3.9
+- mapName: World War II Balanced Mod 2.0
+  url: https://github.com/triplea-maps/world_war_ii_g40_balanced/releases/download/0.10/world_war_ii_g40_balanced.zip
+  description: |
+    <img src="http://tripleamaps.sourceforge.net/images/TripleA_ww2_global_1940_mini.png" />
```
Comment (DanVanAtta) [inline production_config/triplea_maps.yaml]: see if you can host this image file github. When you attach an image to github issues or messages, github will upload it and give you a URL. You can use that URL here. For the thumbnail file, you can also check it into the top level folder of the world_war_ii_g40_balanced repo as well.

**Pull request merged**